modules/validation/helpers.go

```diff
@@ -75,3 +76,21 @@ func IsValidExternalURL(uri string) bool {
 
 	return true
 }
+
+// IsValidExternalTrackerURLFormat checks if URL matches required syntax for external trackers
+func IsValidExternalTrackerURLFormat(uri string) bool {
+	if !IsValidExternalURL(uri) {
+		return false
+	}
+
+	var re = regexp.MustCompile(`({?)(?:user|repo|index)+?(}?)`)
```

Comment:
Can we move this to a package level variable so we compile it only once?

Reply:
sure, moved!